lib/handlers/api/source-controller.ts

```diff
@@ -0,0 +1,42 @@
+import express from 'express';
+import _ from 'underscore';
+
+import {addStaticHeaders} from '../../../app.js';
+import {Source} from '../../../types/source.interfaces.js';
+
+export class SourceController {
```

Comment:
Controller is a very common name used practically everywhere (nest.js, spring boot, dotnet, rails, laravel)

I think it makes more sense to call it controller than handler.

Reply:
Happy to take your advice on this. You have _far_ more experience of real-world web apps than me.